docs/reference/serializers.rst

```diff
@@ -16,6 +16,22 @@ format, namely JSON. This allows us to use the same business logic for the serve
 the client. It also means, that we could create native mobile apps, which communicate with a
 web-application, without ever seeing a line of HTML code.
 
+Moreover, since **django-SHOP** uses **django-CMS** to organize all available components, a classic
+Django "View" does not make much sense anymore. Therefore, as we evolve our Model-View-Control
+pattern into a modern web application, our REST serializers become the new controllers.
+
+
+From a Database Model to the Serializer
+=======================================
+
+As we already know, all database models from the **django-SHOP** framework are owned by the merchant
+implementation. Model serializers reflect their content and hence are tightly coupled with them.
```

Comment:
Not sure if we should call this "tight coupling". Suggestion: Delete the sentence "Model serializers reflect their content and hence are tightly coupled with them.", keep the rest.